### ISSUE TITLE
Add note about putting then and else forms on separate lines in `if`

### ISSUE THREAD

**Issue by camsaul** [bbatsov/clojure-style-guide#243]
I've seen code like 

```clj
(if (or (something)
        (something-else))
  do-something do-something-else)
```

in the wild. I would like to be able to point to a style guide note about this. This doesn't apply of course to things like 

```clj 
(if x then else)
```

Where it's all on one line but if you're going to format it with multiple lines then put the then and else forms on different lines

**Comment by bbatsov** [comment]
Yeah, I agree with that suggestion. Generally I think it's fine to have both branches on the same line only for trivial (e.g. one-line conditionals).

**Comment by seancorfield** [comment]
PR welcome, at this point.

**Comment by seancorfield** [comment]
PR is still welcome :)
